students/drovdahl/lesson08/test_circle.py

```diff
@@ -1,8 +1,8 @@
 #! /usr/local/bin/python3
 '''
-test code for circle.py
+test code for circle2.py
 '''
-from circle import *
+from circle2 import *
```

Comment:
Nice work! Don't forget to change your circle2.py back to circle! But I'm just being annoying. Goood job!